### DISCUSSION TITLE
Taskbar transparant as a feature ?

### DISCUSSION THREAD

**Discussion by streulma** [valinet/ExplorerPatcher#3003] [Ideas]
A friend that uses Explorer Patcher all the day, wants the transparant taskbar as a feature and not as a bug.

![transparanttaskbar](https://github.com/valinet/ExplorerPatcher/assets/6482628/d3ad8316-43fb-49dd-8657-c202de7aee63)

**Comment by IoveBread** [comment]
I saw that today too it looks really cool and I would love it as a feature

**Comment by Amrsatrio** [comment]
Sorry for being dismissive here, but what about installing TranslucentTB? It should take less than a minute to install it from the Microsoft Store.

First, I'd like to say that I'm not sponsored or affiliated in any way with it, but figured that I like it very much with two reasons being lightweight and compatible with EP.

Second, if I add this feature into the Windows 10 taskbar, people would beg me to add stuff like different behaviors during different situations like what TTB can do. As well as making the 11 taskbar transparent, and if they ever stop working they can become a bother for me to maintain. So for now I'm not considering to make this a feature.

Thank you for your understanding.

**Reply by TimMarvelHero** [reply]
well TranslucentTB can't be light mode during the day and dark mode during the evening. It's completely manual. Manually switching the color is kinda tedious. I've seen people make discussions about it on their GitHub and the contributors ignored it. But the switching colors during the day is irrelevant in this discussion.

**Comment by pyrates999** [comment]
This has already been declined as a feature: https://github.com/valinet/ExplorerPatcher/issues/2993
